### PR TITLE
Sourcemaps will no longer minify

### PR DESCRIPTION
**Pull request by adamjgrant**
.map files should no longer have revision numbers

### CODE REVIEW

**Comment by adamjgrant** [comment]
@andrewconner @cvializ

**Comment by andrewconner** [comment]
Cool. Only at concern is that our CDN uploads are set to cache forever. See:

```
    .pipe(parallelize(publisher.publish({'Cache-Control': 'max-age=315360000, no-transform, public'}), 50))
```

which is a great feature. How could we get the updated sourcemaps. Or, could we have them hashed, and dynamically set the sourcemap reference?

**Comment by adamjgrant** [comment]
Will airbrake be looking for this on the CDN?
Le Tue, Jul 7, 2015 à 11:52 PM, Andrew Conner notifications@github.com a
écrit :

**Comment by adamjgrant** [comment]
Nvm. I realized the link is relative. Damn

File: kifi-backend/modules/shoebox/angular/gulpfile.js [hunk 1]
```diff
@@ -443,10 +443,11 @@ gulp.task('test', function (done) {
 });
 
 function compileAssetRevs(opts, dest) {
-  opts = merge({ ignore: [ /^\/favicon.ico$/g, /^sprites\//g ], hashLength: 7 }, opts || {});
+  opts = merge({ dontRenameFile: [ /^\/favicon.ico$/g, /^sprites\//g ], hashLength: 7 }, opts || {});
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/gulpfile.js:446]: Syntax changed in new verison.

**Comment by adamjgrant** [comment]
I think we're good. @andrewconner @cvializ ?

File: kifi-backend/modules/shoebox/angular/gulpfile.js (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/gulpfile.js:446]: Ugh that is the worst when that happens

**Comment by cvializ** [comment]
:+1: looks good to me

**Pull request merged**